### ISSUE TITLE
[Vimeo] Unable to download any video with error: ERROR: Unable to extract info section;

### ISSUE THREAD

**Issue by Linskun** [ytdl-org/youtube-dl#30772]
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dl version **2021.12.17**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
youtube-dl --no-check-certificate --cookies cookies.txt --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) Gecko/20100101 Firefox/99.0" https://player.vimeo.com/video/633805668 --verbose

[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--no-check-certificate', '--cookies', 'cookies.txt', '--user-agent', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:99.0) Gecko/20100101 Firefox/99.0', 'https://player.vimeo.com/video/633805668', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.2 (CPython) - Linux-5.16.14-zen1-1-zen-x86_64-with-glibc2.35
[debug] exe versions: ffmpeg 5.0, ffprobe 5.0, rtmpdump 2.4
[debug] Proxy map: {}
[vimeo] 633805668: Downloading webpage
ERROR: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 815, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 836, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.10/site-packages/youtube_dl/extractor/common.py", line 534, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3.10/site-packages/youtube_dl/extractor/vimeo.py", line 651, in _real_extract
    config = self._parse_json(self._search_regex(
  File "/usr/lib/python3.10/site-packages/youtube_dl/extractor/common.py", line 1012, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract info section; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```

## Description

I'm trying to download **any** video from Vimeo without success.
Here you can see my test with a trailer. I've the same issue with all content.
I've tryed to pass dumped cookies from Firefox/99, pass the user-agent string without success.
Tryed with my Linux and Windows versions

**Comment by dirkf** [comment]
WFM by omitting options, release 2021.12.17 (below) and also git master:
```ShellSession
$ python -m youtube_dl -v -F 'https://player.vimeo.com/video/633805668'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-F', u'https://player.vimeo.com/video/633805668']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: 5014bd67c
[debug] Python version 2.7.17 (CPython) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[vimeo] 633805668: Downloading webpage
[vimeo] 633805668: Downloading akfire_interconnect_quic m3u8 information
[vimeo] 633805668: Downloading akfire_interconnect_quic m3u8 information
[vimeo] 633805668: Downloading fastly_skyfire m3u8 information
[vimeo] 633805668: Downloading fastly_skyfire m3u8 information
[vimeo] 633805668: Downloading akfire_interconnect_quic MPD information
[vimeo] 633805668: Downloading akfire_interconnect_quic MPD information
[vimeo] 633805668: Downloading fastly_skyfire MPD information
[vimeo] 633805668: Downloading fastly_skyfire MPD information
[info] Available formats for 633805668:
format code                                          extension  resolution note
hls-akfire_interconnect_quic_sep-audio-high-audio    mp4        audio only 
hls-akfire_interconnect_quic_sep-audio-medium-audio  mp4        audio only 
hls-fastly_skyfire_sep-audio-high-audio              mp4        audio only 
hls-fastly_skyfire_sep-audio-medium-audio            mp4        audio only 
dash-akfire_interconnect_quic_sep-audio-f1949924     m4a        audio only DASH audio   67k , m4a_dash container, opus  (48000Hz)
dash-fastly_skyfire_sep-audio-f1949924               m4a        audio only DASH audio   67k , m4a_dash container, opus  (48000Hz)
dash-akfire_interconnect_quic_sep-audio-0ff10069     m4a        audio only DASH audio   99k , m4a_dash container, opus  (48000Hz)
dash-fastly_skyfire_sep-audio-0ff10069               m4a        audio only DASH audio   99k , m4a_dash container, opus  (48000Hz)
dash-akfire_interconnect_quic_sep-audio-e1b13e9a     m4a        audio only DASH audio  127k , m4a_dash container, mp4a.40.2 (48000Hz)
dash-fastly_skyfire_sep-audio-e1b13e9a               m4a        audio only DASH audio  127k , m4a_dash container, mp4a.40.2 (48000Hz)
dash-akfire_interconnect_quic_sep-audio-4ce45ea7     m4a        audio only DASH audio  255k , m4a_dash container, mp4a.40.2 (48000Hz)
dash-fastly_skyfire_sep-audio-4ce45ea7               m4a        audio only DASH audio  255k , m4a_dash container, mp4a.40.2 (48000Hz)
dash-akfire_interconnect_quic-video-9e48d6be         mp4        426x240    DASH video  335k , mp4_dash container, avc1.640015, video only
dash-akfire_interconnect_quic_sep-video-9e48d6be     mp4        426x240    DASH video  335k , mp4_dash container, avc1.640015, video only
dash-fastly_skyfire-video-9e48d6be                   mp4        426x240    DASH video  335k , mp4_dash container, avc1.640015, video only
dash-fastly_skyfire_sep-video-9e48d6be               mp4        426x240    DASH video  335k , mp4_dash container, avc1.640015, video only
hls-akfire_interconnect_quic-225                     mp4        426x240     225k , avc1.640015, 23.976fps, video only
hls-fastly_skyfire-225                               mp4        426x240     225k , avc1.640015, 23.976fps, video only
hls-akfire_interconnect_quic_sep-350                 mp4        426x240     350k , avc1.640015, 23.976fps, video only
hls-fastly_skyfire_sep-350                           mp4        426x240     350k , avc1.640015, 23.976fps, video only
dash-akfire_interconnect_quic_sep-video-e1b13e9a     mp4        640x360    DASH video  517k , mp4_dash container, avc1.64001E, video only
dash-fastly_skyfire_sep-video-e1b13e9a               mp4        640x360    DASH video  517k , mp4_dash container, avc1.64001E, video only
hls-akfire_interconnect_quic_sep-506                 mp4        640x360     506k , avc1.64001E, 23.976fps, video only
hls-fastly_skyfire_sep-506                           mp4        640x360     506k , avc1.64001E, 23.976fps, video only
dash-akfire_interconnect_quic_sep-video-4ce45ea7     mp4        960x540    DASH video 1452k , mp4_dash container, avc1.64001F, video only
dash-fastly_skyfire_sep-video-4ce45ea7               mp4        960x540    DASH video 1452k , mp4_dash container, avc1.64001F, video only
hls-akfire_interconnect_quic_sep-991                 mp4        960x540     991k , avc1.64001F, 23.976fps, video only
hls-fastly_skyfire_sep-991                           mp4        960x540     991k , avc1.64001F, 23.976fps, video only
dash-akfire_interconnect_quic_sep-video-ba88b87d     mp4        1280x720   DASH video 2720k , mp4_dash container, avc1.640020, video only
dash-fastly_skyfire_sep-video-ba88b87d               mp4        1280x720   DASH video 2720k , mp4_dash container, avc1.640020, video only
hls-akfire_interconnect_quic_sep-1874                mp4        1280x720   1874k , avc1.640020, 23.976fps, video only
hls-fastly_skyfire_sep-1874                          mp4        1280x720   1874k , avc1.640020, 23.976fps, video only
dash-akfire_interconnect_quic_sep-video-e4723635     mp4        1920x1080  DASH video 5420k , mp4_dash container, avc1.640028, video only
dash-fastly_skyfire_sep-video-e4723635               mp4        1920x1080  DASH video 5420k , mp4_dash container, avc1.640028, video only
hls-akfire_interconnect_quic_sep-4269                mp4        1920x1080  4269k , avc1.640028, 23.976fps, video only
hls-fastly_skyfire_sep-4269                          mp4        1920x1080  4269k , avc1.640028, 23.976fps, video only
dash-akfire_interconnect_quic_sep-video-ad35ac86     mp4        2560x1440  DASH video 12574k , mp4_dash container, avc1.640032, video only
dash-fastly_skyfire_sep-video-ad35ac86               mp4        2560x1440  DASH video 12574k , mp4_dash container, avc1.640032, video only
hls-akfire_interconnect_quic_sep-9445                mp4        2560x1440  9445k , avc1.640032, 23.976fps, video only
hls-fastly_skyfire_sep-9445                          mp4        2560x1440  9445k , avc1.640032, 23.976fps, video only
dash-akfire_interconnect_quic_sep-video-6897c527     mp4        3840x2160  DASH video 23230k , mp4_dash container, avc1.640034, video only
dash-fastly_skyfire_sep-video-6897c527               mp4        3840x2160  DASH video 23230k , mp4_dash container, avc1.640034, video only
hls-akfire_interconnect_quic_sep-18261               mp4        3840x2160  18261k , avc1.640034, 23.976fps, video only
hls-fastly_skyfire_sep-18261                         mp4        3840x2160  18261k , avc1.640034, 23.976fps, video only
dash-akfire_interconnect_quic-video-e1b13e9a         mp4        640x360    DASH video  644k , mp4_dash container, avc1.64001E, mp4a.40.2 (48000Hz)
dash-fastly_skyfire-video-e1b13e9a                   mp4        640x360    DASH video  644k , mp4_dash container, avc1.64001E, mp4a.40.2 (48000Hz)
http-360p                                            mp4        640x360    23fps
hls-akfire_interconnect_quic-506                     mp4        640x360     506k , avc1.64001E, 23.976fps, mp4a.40.2
hls-fastly_skyfire-506                               mp4        640x360     506k , avc1.64001E, 23.976fps, mp4a.40.2
dash-akfire_interconnect_quic-video-4ce45ea7         mp4        960x540    DASH video 1707k , mp4_dash container, avc1.64001F, mp4a.40.2 (48000Hz)
dash-fastly_skyfire-video-4ce45ea7                   mp4        960x540    DASH video 1707k , mp4_dash container, avc1.64001F, mp4a.40.2 (48000Hz)
http-540p                                            mp4        960x540    23fps
hls-akfire_interconnect_quic-1120                    mp4        960x540    1120k , avc1.64001F, 23.976fps, mp4a.40.2
hls-fastly_skyfire-1120                              mp4        960x540    1120k , avc1.64001F, 23.976fps, mp4a.40.2
dash-akfire_interconnect_quic-video-ba88b87d         mp4        1280x720   DASH video 2976k , mp4_dash container, avc1.640020, mp4a.40.2 (48000Hz)
dash-fastly_skyfire-video-ba88b87d                   mp4        1280x720   DASH video 2976k , mp4_dash container, avc1.640020, mp4a.40.2 (48000Hz)
http-720p                                            mp4        1280x720   23fps
hls-akfire_interconnect_quic-1874                    mp4        1280x720   1874k , avc1.640020, 23.976fps, mp4a.40.2
hls-fastly_skyfire-1874                              mp4        1280x720   1874k , avc1.640020, 23.976fps, mp4a.40.2
dash-akfire_interconnect_quic-video-e4723635         mp4        1920x1080  DASH video 5675k , mp4_dash container, avc1.640028, mp4a.40.2 (48000Hz)
dash-fastly_skyfire-video-e4723635                   mp4        1920x1080  DASH video 5675k , mp4_dash container, avc1.640028, mp4a.40.2 (48000Hz)
http-1080p                                           mp4        1920x1080  23fps
hls-akfire_interconnect_quic-4269                    mp4        1920x1080  4269k , avc1.640028, 23.976fps, mp4a.40.2
hls-fastly_skyfire-4269                              mp4        1920x1080  4269k , avc1.640028, 23.976fps, mp4a.40.2
dash-akfire_interconnect_quic-video-ad35ac86         mp4        2560x1440  DASH video 12829k , mp4_dash container, avc1.640032, mp4a.40.2 (48000Hz)
dash-fastly_skyfire-video-ad35ac86                   mp4        2560x1440  DASH video 12829k , mp4_dash container, avc1.640032, mp4a.40.2 (48000Hz)
hls-akfire_interconnect_quic-9446                    mp4        2560x1440  9446k , avc1.640032, 23.976fps, mp4a.40.2
hls-fastly_skyfire-9446                              mp4        2560x1440  9446k , avc1.640032, 23.976fps, mp4a.40.2
dash-akfire_interconnect_quic-video-6897c527         mp4        3840x2160  DASH video 23485k , mp4_dash container, avc1.640034, mp4a.40.2 (48000Hz)
dash-fastly_skyfire-video-6897c527                   mp4        3840x2160  DASH video 23485k , mp4_dash container, avc1.640034, mp4a.40.2 (48000Hz)
hls-akfire_interconnect_quic-18261                   mp4        3840x2160  18261k , avc1.640034, 23.976fps, mp4a.40.2
hls-fastly_skyfire-18261                             mp4        3840x2160  18261k , avc1.640034, 23.976fps, mp4a.40.2 (best)
$
```

**Comment by Linskun** [comment]
Ok issue closed.

I've investigated on network side and found that the problem was on DNS server !
Sorry for the wrong reporting !
